src/DefaultProxyUnleash.php

```diff
@@ -54,4 +55,13 @@ public function getVariant(string $featureName, ?Context $context = null, ?Varia
 
         return $variant;
     }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    #[Override]
+    public function refreshCache(): void
+    {
+        throw new BadMethodCallException('This method is not supported in the proxy implementation');
```

Comment:
Ideally I'd like to see no changes to this file. Having a method that breaks at runtime doesn't feel good to me, I think we can move this method down into the Repository perhaps, at least that way, it's clear to end users that this isn't something you can do with the Proxy instance

Reply:
Resolving this due to the other conversation where we decided to go in a different direction.

Reply:
Removed the changes to the Unlesh interface